.idea/workspace.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure this file needs to be in the PR. Maybe something to add to .gitignore.

Reply:
I don't think you should commit .gitignore though @stevenctl - we don't have one yet at least?

Reply:
Why not start one?

Reply:
No strong opinion, just that this repo is used for different things.  Could we [inherit from istio/istio](https://github.com/istio/istio/blob/master/.gitignore)?